packages/webview_flutter/webview_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 4.8.1
+
+* Adds support for detecting web view render process termination on Android via navigation delegate. See
+  `NavigationDelegate.onWebViewRenderProcessTerminated`.
+  Calling application should return `true` if termination was handled by the app, otherwise `false`.
+  For more details see: https://developer.android.com/reference/android/webkit/WebViewClient#onRenderProcessGone(android.webkit.WebView,%20android.webkit.RenderProcessGoneDetail)
```

Comment:
We do not generally want to put Android-specific docs in the app-facing package. Our cross-platform abstraction needs to be understandable without going to platform-level docs.